_board/beetle-esp32-c3.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: download
+board_id: "beetle-esp32-c3"
+title: "DFRobot Beetle ESP32-C3 Download"
+name: "DFRobot Beetle ESP32-C3"
+manufacturer: "DFRobot"
+board_url: "https://wiki.dfrobot.com/SKU_DFR0868_Beetle_ESP32_C3"
+board_image: "beetle-esp32-c3.png"
+date_added: 2022-07-17
+downloads_display: true
+blinka: false
+family: esp32c3
+bootloader_id: lolin_c3_mini
+features:
+  - Arduino Shield Compatible
```

Comment:
Arduino Shield Compatible means the board is shaped like https://www.adafruit.com/product/2488 and a shield like https://www.adafruit.com/product/94 would fit on top.

Reply:
```suggestion
```

Thanks for catching that. I'm not Arduino knowledgeable and I had meant to look into that to verify.